tfjs-converter/src/executor/graph_executor.ts

```diff
@@ -225,12 +264,16 @@ export class GraphExecutor implements FunctionExecutor {
           this.weightMap, tensorArrayMap, tensorListMap,
           this.functionExecutorMap);
       const tensorsMap: NamedTensorsMap = {...this.weightMap};
+      if (this.keepIntermediateTensors) {
+        this.clonedTensorsMap = this.cloneTensorMap(this.weightMap);
+      }
 
       Object.keys(inputs).forEach(name => {
         const [nodeName, index] = parseNodeName(name);
         const tensors: Tensor[] = [];
         tensors[index] = inputs[name];
         tensorsMap[nodeName] = tensors;
+        this.cloneTensorList(tensors, nodeName, this.clonedTensorsMap);
```

Comment:
Nit: Make `this.cloneTensorsList` return a cloned list of tensors instead of storing them in a map. I think this is a bit easier to read, and it matches `this.cloneTensorsMap`. Also, then `this.cloneTensorsMap` can use `this.cloneTensorsList` in its implementation.

```suggestion
        this.clonedTensorsMap[nodeName] = this.cloneTensorList(tensors);
```

Reply:
done